Sources/Fullscreen/Explore/Helpers/ExploreLayoutBuilder.swift

```diff
@@ -87,6 +89,25 @@ private extension NSCollectionLayoutSection {
         ),
         scrollingBehavior: .continuous
     )
+    
+    static let banner = {
+        let group = NSCollectionLayoutGroup.horizontal(
+            layoutSize: NSCollectionLayoutSize(
+                widthDimension: .fractionalWidth(1),
+                heightDimension: .estimated(10)
```

Comment:
We should set the estimated number to something closer to the actual estimated height of the banner, for better performance. What about using the height of a typical banner with 1 LOC in the title, and 2 LOC in the text, or something like that? Or whatever you think the average banner looks like 🤔

Reply:
I made an estimate based on the current one, because it seems like they are mostly using a similar structure for their designs.